src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactNode } from "react"
+import styles from "./Breadcrumb.module.css"
+import { BreadcrumbItem } from "./BreadcrumbItem"
+
+export interface Link {
+  href: string
+  label: string
+  onClick?: () => void
+}
+
+export type RenderItem = ({
+  item,
```

Comment:
A bit of a nit-pick ⛏️. Would `link` maybe be a better name here? It feels weird to me that you have a `renderItem` callback that returns an object with an `item` property.

Reply:
That said, I guess we are rendering an `item` and said item is given to us here for us to then render how we wish. Maybe just ignore me altogether. Ha.

Reply:
@thispastwinter I was trying to keep the name of this fairly generic since the item rendered can really be anything - a button, router link, anchor tag, etc.

Reply:
Yeah, totally. It makes more sense to me now and feels less weird now. I think I just saw how the item's type was assigned to your `Link` type def and it got me thinking about whether it should be called item or not.